documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -17,21 +20,22 @@ The procedure describes the renewal of CA certificates in PEM format.
 
 * The Cluster Operator is running.
 * xref:installing-your-own-ca-certificates-{context}[Your own CA certificates and private keys are installed].
-* You have new cluster or clients X.509 certificates in PEM format.
+* You have new cluster or client X.509 certificates in PEM format.
```

Comment:
It's actually "clients" ... as we call it (it's the CA certificate to sign clients related certs)